Source/Core/FeatureDetection.js

```diff
@@ -255,6 +253,71 @@ define([
         }
     }
 
+    var supportsCreateImageBitmapResult;
+    function supportsCreateImageBitmap() {
+        if (!defined(supportsCreateImageBitmapResult)) {
+            supportsCreateImageBitmapResult = defined(window.createImageBitmap);
+        }
+
+        return supportsCreateImageBitmapResult;
+    }
+
+    var supportsImageBitmapOptionsResult;
+    var supportsImageBitmapOptionsPromise;
+    function supportsImageBitmapOptions() {
+        // Until the HTML folks figure out what to do about this, we need to actually try loading an image to
+        // know if this browser supports passing options to the createImageBitmap function.
+        // https://github.com/whatwg/html/pull/4248
+        if (defined(supportsImageBitmapOptionsPromise)) {
+            return supportsImageBitmapOptionsPromise.promise;
+        }
+
+        supportsImageBitmapOptionsPromise = when.defer();
+
+        if (!supportsCreateImageBitmap()) {
+            supportsImageBitmapOptionsResult = false;
+            supportsImageBitmapOptionsPromise.resolve(supportsImageBitmapOptionsResult);
+            return supportsImageBitmapOptionsPromise.promise;
+        }
+
+        var imageDataUri = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVQImWP4////fwAJ+wP9CNHoHgAAAABJRU5ErkJggg==';
+        fetch(imageDataUri)
```

Comment:
Why is this using the `fetch` API instead of `Resource.fetchBlob`?  Same question for where `fetch` is used in one spot in `Resource.js`.  In general we don't use the `fetch` API in Cesium because of lacking IE11 support.  That can/will change eventually, but is there a good reason for it here?

Reply:
I needed a `Blob` because `createImageBitmap` blocks the main thread if it's given an `HTMLImageElement`. But if I can do that without `fetch` then I don't think I need it. I'll switch that and confirm that the behavior hasn't changed. Thanks for pointing this out.